src/db/migrations/00012.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE cluster ADD COLUMN last_update TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now();
+ALTER TABLE cluster ADD COLUMN enabled BOOLEAN NOT NULL DEFAULT TRUE;
```

Comment:
what's now the difference between active and enabled? Isn't one of them sufficient?

Reply:
active is cluster status set by checker automatically.

enabled is set manually, it's useful when we need to disable a cluster for a short time.